specification/managementgroups/resource-manager/Microsoft.Management/preview/2024-01-01-preview/management.json

```diff
@@ -0,0 +1,2383 @@
+          "x-nullable": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagementGroupPathElement"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        }
+      }
+    },
+    "ManagementGroupChildInfo": {
+      "description": "The child information of a management group.",
+      "properties": {
+        "type": {
+          "title": "The type of child resource.",
+          "$ref": "#/definitions/ManagementGroupChildType",
+          "description": "The fully qualified resource type which includes provider namespace (e.g. Microsoft.Management/managementGroups)"
+        },
+        "id": {
+          "type": "string",
+          "description": "The fully qualified ID for the child resource (management group or subscription).  For example, /providers/Microsoft.Management/managementGroups/0000000-0000-0000-0000-000000000000"
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the child entity."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The friendly name of the child resource."
+        },
+        "children": {
+          "description": "The list of children.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagementGroupChildInfo"
+          }
+        }
+      }
+    },
+    "ManagementGroupPathElement": {
+      "description": "A path element of a management group ancestors.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the group."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The friendly name of the group."
+        }
+      }
+    },
+    "ParentGroupInfo": {
+      "description": "(Optional) The ID of the parent management group.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The fully qualified ID for the parent management group.  For example, /providers/Microsoft.Management/managementGroups/0000000-0000-0000-0000-000000000000"
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the parent management group"
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The friendly name of the parent management group."
+        }
+      }
+    },
+    "ManagementGroupChildType": {
+      "type": "string",
+      "enum": [
+        "Microsoft.Management/managementGroups",
+        "/subscriptions"
+      ],
+      "x-ms-enum": {
+        "name": "ManagementGroupChildType",
+        "modelAsString": false,
+        "values": [
+          {
+            "value": "Microsoft.Management/managementGroups"
+          },
+          {
+            "value": "/subscriptions"
+          }
+        ]
+      },
+      "description": "The type of child resource."
+    },
+    "OperationResults": {
```

Comment:
>OperationResults [](http://example.com/codeflow?start=5&length=16)

This doesn't seem to be used anywhere. If you end up needing something like this, try using OperationStatusResult from v5 common types